plutus-core/plutus-ir/test/prettyprintingReadable/even3.golden

```diff
@@ -0,0 +1,14 @@
+letrec data Nat | match_Nat where
+         Zero : Nat
+         Suc : Nat -> Nat
+in let data Bool | match_Bool where
```

Comment:
What do you think about this layout?
```
let ....
in
let ...
in
...
```
Pros: `let` always appears at the start of the line, easier to scan
Cons: more lines

Reply:
Other than being a bit (a lot) unorthodox I can't say I particularly mind it. Your call.

Reply:
I think we can keep the orthodox version, it just occurred to me!